INSTALL.md

```diff
@@ -63,207 +63,286 @@ cfgov assets
 cfgov start django
 ```
 
+
 ## Stand-alone installation
```

Comment:
I could reverse them, and also add a note to the Vagrant section that it's not currently working, YMMV, etc.?